com.unity.multiplayer.mlapi/Runtime/Messaging/MessageQueue/MessageFrameItem.cs

```diff
@@ -1,28 +1,30 @@
 using System;
+using System.IO;
+using MLAPI.Serialization;
 using MLAPI.Transports;
 using MLAPI.Serialization.Pooled;
 
 namespace MLAPI.Messaging
 {
     /// <summary>
-    /// FrameQueueItem
-    /// Container structure for RPCs written to the Queue Frame
-    /// Used for both Inbound and Outbound RPCs
+    /// MessageFrameItem
+    /// Container structure for messages written to the Queue Frame
+    /// Used for both Inbound and Outbound messages
     /// NOTE: This structure will change in the near future and is in a state of flux.
     /// This will include removing specific properties or changing property types
     /// </summary>
-    internal struct RpcFrameQueueItem
+    internal struct MessageFrameItem
     {
         public NetworkUpdateStage UpdateStage;
-        public RpcQueueContainer.QueueItemType QueueItemType;
-        public ulong NetworkId; //Sender's network Identifier
+        public MessageQueueContainer.MessageType MessageType;
+        public ulong NetworkId; //Sender's network Identifier, or recipient identifier for server RPCs
```

Comment:
These comments I would either remove or put in a xml doc field.

Reply:
+1